fiftyone/utils/cvat.py

```diff
@@ -6498,14 +6498,21 @@ def _create_detection_shapes(
                     continue
 
                 if self._server_version >= Version("2.3"):
-                    x, y, _, _ = det.bounding_box
+                    x, y, w, h = det.bounding_box
                     frame_width, frame_height = frame_size
                     mask_height, mask_width = det.mask.shape
                     xtl, ytl = round(x * frame_width), round(y * frame_height)
-                    xbr, ybr = xtl + mask_width, ytl + mask_height
+                    w, h = round(w * frame_width), round(h * frame_height)
+                    xbr, ybr = xtl + w, ytl + h
 
                     # -1 to convert from CVAT indexing
-                    rle = HasCVATBinaryMask._mask_to_cvat_rle(det.mask)
+                    mask = det.mask
+                    if w != mask_width or h != mask_height:
+                        mask = etai.resize(
```

Comment:
`_mask_to_cvat_rle()` currently assumes it is provided a binary mask. Just to be safe, I'd recommend adding `.astype("bool")` to cast `mask` back to boolean after resizing.

Or you could change [this](https://github.com/voxel51/fiftyone/blob/7b1e0a0ac778fffecfa17d501b713dcf5ba5d53b/fiftyone/utils/cvat.py#L1609) to `value >= 1`.